src/NovelRT/Graphics/Vulkan/VulkanGraphicsAdapterSelector.cpp

```diff
@@ -55,14 +55,21 @@ namespace NovelRT::Graphics::Vulkan
 
         SwapChainSupportDetails supportDetails = Utilities::QuerySwapChainSupport(physicalDevice, surfaceContext);
         QueueFamilyIndices indices = Utilities::FindQueueFamilies(physicalDevice, surfaceContext);
-
+#ifndef __APPLE__
         if (deviceFeatures.geometryShader == VK_FALSE || !indices.IsComplete() ||
             !CheckPhysicalDeviceRequiredExtensionSupport(physicalDevice) || supportDetails.formats.empty() ||
             supportDetails.presentModes.empty())
         {
             score = -1;
         }
-
+#else
+        if (!indices.IsComplete() ||
+            !CheckPhysicalDeviceRequiredExtensionSupport(physicalDevice) || supportDetails.formats.empty() ||
+            supportDetails.presentModes.empty())
+        {
+            score = -1;
+        }
+#endif
```

Comment:
`deviceFeatures.geometryShader == VK_FALSE` causes the check to fail immediately as the particular device I have does not support a geometry shader. However, I cannot test on a newer Apple device to see if it's just a MoltenVK thing or not, therefore until we can properly test it I've just ifdef'ed that one check. Checks out atm and renders, but just something to keep in mind.

Reply:
I'm not sure we need geometry shader support. This might be a leftover from other things. Please test this on Windows.